recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Please don't increase the build numbers, it's not really necessary.

Reply:
Yes, I know now.
Shall I do a new PR ?

Reply:
Don't worry, I can do these changes myself.